ush/python/pygfs/jedi/jedi.py

```diff
@@ -216,18 +218,45 @@ def get_bias_dict(self, task_config: AttrDict) -> Dict[str, Any]:
                 obfile = ob['obs bias']['input file']
                 obdir = os.path.dirname(obfile)
                 basename = os.path.basename(obfile)
-                prefix = '.'.join(basename.split('.')[:-2])
-                for file in ['satbias.nc', 'satbias_cov.nc', 'tlapse.txt']:
-                    bfile = f"{prefix}.{file}"
-                    copylist.append([os.path.join(task_config.COM_ATMOS_ANALYSIS_PREV, bfile), os.path.join(obdir, bfile)])
-                    # TODO: Why is this specific to ATMOS?
+                prefix = '.'.join(basename.split('.')[:-3])
+                bfile = f"{prefix}.{bias_file}"
+                tar_file = os.path.join(obdir, bfile)
+                copylist.append([os.path.join(task_config.VarBcDir, bfile), tar_file])
+                break
 
         bias_dict = {
             'mkdir': [os.path.join(task_config.DATA, 'bc')],
             'copy': copylist
         }
+
         return bias_dict
 
+    @logit(logger)
+    def extract_tar(self, task_config: AttrDict, tar_dict) -> Dict[str, Any]:
+        """Extract files from list of tarfiles
+
+        This method extract bias correction files from tarball(s)
+
+        Parameters
+        ----------
+        task_config: AttrDict
+            Attribute-dictionary of all configuration variables associated with a GDAS task.
+        tar_dict
+            a dictionary containing the list of tar files
+
+        Returns
+        ----------
+        None
+        """
+
+        # extract bias correction files from tar file
+        for tar_file in tar_dict['copy']:
+            if ".tar" in tar_file[1]:
```

Comment:
What does `tar_dict` actually look like? This index is confusing me; why the 1st index and not the 0th?

Also not sure why this filters by the file extension. Why is it in the tar_dict if it isn't a tarfile?

Reply:
Previous questions aside, even if this is what you want, if you just want to check the file extension, you should use `endswith()` to make sure you only pick up matches at the end of the filename:
```suggestion
            if tar_file[1].endswith('.tar'):
```

Reply:
Below is an example of runtime printout showing the contents of `tar_dict`
```
{'mkdir': ['/scratch1/NCEPDEV/stmp2/Russ.Treadon/RUNDIRS/prjedi/gdas.2024022400/gdasatmanl_00/bc'], 'copy': [['/scratch1/NCEPDEV/stmp2/Russ.Treadon/COMROOT/prjedi/gdas.20240223/18//analysis/atmos/gdas.t18z.rad_varbc_params.tar', '/scratch1/NCEPDEV/stmp2/Russ.Treadon/RUNDIRS/prjedi/gdas.2024022400/gdasatmanl_00/obs/gdas.t18z.rad_varbc_params.tar']]}
```

`tar_file[1]` is referenced because this is the destination of the copy.  The tarball is copied to the job run directory .  We want to untar the tarball in the job run directory.  Subsequent lines reference `tar_file[1]`.

Suggestion to use `endswith` committed at eb4fc374.

Reply:
Ah, my brain isn't working well today I guess; I should've known that. Still not sure why the filter on `*.tar` is needed. Will non-tar files wind up in the dict?

Reply:
This PR moves the radiance bias correction files into a tarball.  Aircraft bias correction uses a single file, not a tarball.  If the bias dictionary contains both the radiance and aircraft bias correction files, we only want `extract_tar` to act on the radiance tarball.

Maybe I'm being overly cautious.

Reply:
@WalterKolczynski-NOAA , any changes you want to see here?